src/Public.Api/Extract/ExtractController.cs

```diff
@@ -0,0 +1,100 @@
+namespace Public.Api.Extract
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Be.Vlaanderen.Basisregisters.Api;
+    using Be.Vlaanderen.Basisregisters.Api.Exceptions;
+    using Common.Infrastructure;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Extensions.Configuration;
+    using Microsoft.Extensions.Logging;
+    using Newtonsoft.Json.Converters;
+    using Swashbuckle.AspNetCore.Filters;
+    using ProblemDetails = Be.Vlaanderen.Basisregisters.BasicApiProblem.ProblemDetails;
+    
+    [ApiVersion("1.0")]
+    [AdvertiseApiVersions("1.0")]
+    [ApiRoute("")]
+    [ApiExplorerSettings(GroupName = "Extract")]
+    [Produces(AcceptTypes.Json, AcceptTypes.JsonLd, AcceptTypes.Xml)]
```

Comment:
This is probably something different, check if there is a produces for multipart mimetype? or whatever a zip file from s3 represents